scripts/setup-k8s-secrets.sh

```
@@ -0,0 +1,258 @@
+    read -p "Enter Anthropic API Key: " -s ANTHROPIC_KEY
+    echo
+    read -p "Enter SERP API Key: " -s SERP_KEY
+    echo
+    read -p "Enter Bing Search API Key: " -s BING_KEY
+    echo
+    
+    # Update secrets
+    kubectl patch secret api-keys \
+        --namespace="$NAMESPACE" \
+        --type='json' \
+        -p='[{"op": "replace", "path": "/data/openai-api-key", "value": "'$(echo -n "$OPENAI_KEY" | base64)'"}]'
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)